.github/workflows/smoke-test.yml

```diff
@@ -31,6 +31,10 @@ jobs:
           sudo snap refresh
           sudo snap list
 
+          # get the latest beta snapd (to satisfy our "assumes:")
+          sudo snap refresh --beta snapd
+          # TODO remove "--beta" once 2.50 hits stable
```

Comment:
Not sure what to do about this -- maybe we wait to merge this until 2.50 is stable?

Reply:
... oh, 2.50 hit candidate yesterday, so it's probably a short time before it's stable anyhow :smile: :innocent:

Reply:
plan is for 2.50 to hit stable on may 10th, we have a company-wide sprint next week, hence the week after